tidb-cloud/terraform-provider-tidbcloud.md

```diff
@@ -0,0 +1,1343 @@
+---
+title: TiDB Cloud Terraform
+summary: TiDB Cloud Terraform
+---
+
+# Terraform TiDB Cloud Provider
+
+terraform-provider-tidbcloud allows one to use Terraform with TiDB Cloud. Learn more about [TiDB Cloud](https://en.pingcap.com/tidb-cloud/)
+
+For general information about Terraform, visit the [official website](https://www.terraform.io) and the [GitHub project page](https://github.com/hashicorp/terraform).
```

Comment:
This is the official site of our terraform provider, so don't direct users to other places. And we are at tidb cloud doc, so no need to learn more.

Reply:
Get